ansible/roles/django/tasks/main.yml

```
@@ -0,0 +1,110 @@
+
+- name: Set up cron jobs for Django management commands
+  cron:
+    name: "{{ item.name }}"
+    minute: "{{ item.minute }}"
+    hour: "{{ item.hour }}"
+    day: "{{ item.day | default('*') }}"
+    weekday: "{{ item.weekday | default('*') }}"
+    job: "cd {{ project_path }} && {{ project_path }}/venv/bin/python manage.py {{ item.command }} >> {{ project_path }}/logs/cron.log 2>&1"
+    user: "{{ project_user }}"
+  loop:
+    - { name: "Send session reminders", minute: "0", hour: "*", command: "send_session_reminders" }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)